npm-search/assets/js/app.js

```diff
@@ -168,7 +168,15 @@ var Plugin = React.createClass({
             </li>
         )
     }
-})
+});
+
+var timer=0;
```

Comment:
We could have added it as a state variable to the plugin list component, but that would unnecessarily complicate stuff.